### PR TITLE
TCI-1062 Anchor link missing in sanmateo texteditor added.

### PR DESCRIPTION
**Pull request by reshmai**
Please review.

### CODE REVIEW

File: web/profiles/jcc_components_profile/jcc_components_profile.info.yml
Comment (melwong-jcc) [inline web/profiles/jcc_components_profile/jcc_components_profile.info.yml:1]: Though this should ensure the module is installed especially for new sites, the core.extension.yml will eventually need to be updated per existing site using immutable to manage their core configs.

**Review by melwong-jcc — APPROVED** [review]
Approved for develop testing but consider updating all existing core.extension.yml files now that we are making the feature global.

File: web/profiles/jcc_components_profile/jcc_components_profile.info.yml
Comment (reshmai) [reply web/profiles/jcc_components_profile/jcc_components_profile.info.yml:1]: I have enabled anchor_link module for remaining sites. Please review.

**Review by melwong-jcc — COMMENTED** [review]
Interesting. I guess I did apply to all TC sites at the time the module was added so the missing piece was the immutable part. Sorry for the extra work but doesn't hurt to enable to all sites.

**Pull request merged**